src/index.js

```diff
@@ -4,10 +4,8 @@
   jsx-a11y/click-events-have-key-events: 0 */
 import React, { Component, Children, cloneElement } from 'react';
 import PropTypes from 'prop-types';
-import nth from 'lodash.nth';
-import merge from 'lodash.merge';
+import { nth, merge } from 'lodash';
```

Comment:
Did you explore completely getting rid of `merge`?
From what I can see you could use a flat merge on all usages of merge on this file (plain inline-style objects, and this one https://github.com/godaddy/react-img-carousel/blob/master/src/index.js#L252 never updates the values on the same `imageSrc` inside `loadedImages`)

Reply:
`nth` could be worked around as well it seems like, and it might be even causing unexpected behavior when the index is negative at: https://github.com/godaddy/react-img-carousel/blob/master/src/index.js#L644-L645 (I could be wrong, didn't look at the full logic)